src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background-color: $white;
+  border: 1px solid $borderColor;
+  border-radius: 5px;
+
+  &--container {
```

Comment:
Will be better to make the card as an element of catalog, and move margins to those class
Instead of creating several containers for one card